frontend/app/src/pages/role-management/index.tsx

```diff
@@ -1,12 +1,35 @@
+import { GLOBAL_PERMISSION_OBJECT } from "@/config/constants";
+import useQuery from "@/hooks/useQuery";
+import ErrorScreen from "@/screens/errors/error-screen";
+import UnauthorizedScreen from "@/screens/errors/unauthorized-screen";
 import Content from "@/screens/layout/content";
+import LoadingScreen from "@/screens/loading-screen/loading-screen";
+import { getObjectPermissionsQuery } from "@/screens/permission/queries/getObjectPermissions";
 import { RoleManagementNavigation } from "@/screens/role-management";
+import { gql } from "@apollo/client";
 import { Outlet } from "react-router-dom";
 
 function RoleManagement() {
+  const { loading, error } = useQuery(gql(getObjectPermissionsQuery(GLOBAL_PERMISSION_OBJECT)));
```

Comment:
why? We do not even use the data from the query

Reply:
it's to check the permissions for the role management views, we only need to check the 403 error to prevent the access to the role management, more data is used in the different views if we have the view permission

the goal is to test the view access in one point, to block the whole view instead of using the permissions in the list views with the header and tabs